pages/atom.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+import Layout from "../components/layout/layout";
+import ContentSection from "../components/content-section/content-section";
+import React from "react";
+
+import {
+  PageHeader,
+  PageHeaderLeft,
+  PageHeaderRight
+} from "../components/page-header/page-header";
+
+const InteractiveCoding = () => (
+  <ContentSection>
+    <div className="panes center-vertically">
+      <div className="pane-70 pane">
+        <div className="section-graphic">
+          <video
+            style={{ boxShadow: "0 4px 14px 0 rgba(0,0,0,.1)" }}
+            autoPlay
+            loop
+          >
+            <source src="static/atom/interactive.mp4" type="video/mp4" />
```

Comment:
The videos have super high resolution (2048 × 1280) which may be overkill but they have been compressed to have a reasonable file size (100-300 kB).

@rgbkrk Experienced weird anti-aliasing issues if the browser window is narrow, possibly caused by the high resolution.
I could scale down the videos but then we would loose the sharpness on HighDPI screens.

HTML5 on modern browsers supports [media queries](https://www.iandevlin.com/blog/2012/08/html5/responsive-html5-video) which allow to add multiple versions for different resolutions. There are also [shims](http://blog.etianen.com/blog/2013/05/19/html5-video-is-hard/) for legacy browsers.

@aulneau Is there canonical of handling this? Or should we just use lower res videos?